target/linux/layerscape/image/armv8_64b.mk

```diff
@@ -199,7 +213,8 @@ endef
 TARGET_DEVICES += ls1088ardb-sdboot
 
 define Device/ls2088ardb
-  DEVICE_TITLE := LS2088ARDB
+  DEVICE_VENDOR := NXP
+  DEVICE_MODEL := LS2088ARDB
```

Comment:
Add dash before RDB suffix.

Reply:
This one seems to be the only exception where the company refers to it without dash.

Reply:
yes, an odd one.